include/mlir/SPIRV/SPIRVTypes.h

```diff
@@ -52,9 +53,28 @@ enum Kind {
 };
 }
 
+// Provides a common base class between SPIR-V ArrayType, SPIR-V StructType.
```

Comment:
`... for VectorType, SPIR-V ArrayType, SPIR-V StructType`.

Reply:
Updated.